### PR TITLE
Create sales employee table 1

### PR DESCRIPTION
**Pull request by Criofann**
Adding the first set of scripts to the main directory so the rest of the team can work on it.
This addition allows for the creation of the create sales employee table as well as some example data that is being entered into the table for testing purposes

Proposed solution for point 1 of the product backlog "As a member of the HR team I want to be able to create a new sales employee. I should be able to store a name, salary, bank account number, national insurance number and commission rate"

### CODE REVIEW

File: CreateSalesEmployeeTable_1.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+	-- Create the Sales_Employee table
+    CREATE TABLE Sale_Employee (
+	SaleEmployeeID int AUTO_INCREMENT PRIMARY KEY,
+    FirstName Varchar(100),
+    LastName Varchar(100),
+    Salary Decimal(10,2),
+    BankAccount varchar(16),
+    NationalInsurance varchar(11)
+	);
```
Comment (jkelso662) [inline CreateSalesEmployeeTable_1.sql:16]: Indentation again. Also be consistent with your use of caps! E.g. varchar and Varchar

File: README.md [hunk 2]
```diff
@@ -1 +0,0 @@
-#jcac-Softworn
```
Comment (jkelso662) [inline README.md:1]: Why did you delete your README.md file?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nearly there, couple of small things to change!

File: CreateSalesEmployeeTable_1.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
+USE JCAC_CormacF
+DELIMITER $$
+DROP PROCEDURE IF EXISTS CreateSalesEmployeeTable_1 $$
+CREATE PROCEDURE CreateSalesEmployeeTable_1()
```
Comment (jkelso662) [inline CreateSalesEmployeeTable_1.sql:4]: Your procedure name doesn't look to be the same format as merge request 4... Could you check what convention your team want to go for?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Check procedure names so I can approve. Again, few wee indentation issues please try to keep ontop of these but won't block just on indentation issue.

**Review by Joseph-Magee — APPROVED** [review]
